src/cmd/root.go

```diff
@@ -50,6 +50,11 @@ func runProjectCmd(args []string) {
 		_ = logFile.Close()
 	}()
 	runner := getProjectRunner(args, *pcFlags.NoDependencies, "", []string{})
+	if *pcFlags.DryRun {
+		processNames, _ := runner.GetLexicographicProcessNames()
+		fmt.Printf("Loaded %d processes from %d config files.\n", len(processNames), len(opts.FileNames)+len(opts.EnvFileNames))
```

Comment:
Do you think that "Loaded" really reflects the intent of this feature?
Maybe something like "Successfully validated" will make more sense for the `--dry-run` flag?

Reply:
Yes, I can do that.